.github/workflows/ttnn-run-sweeps.yaml

```diff
@@ -244,9 +247,10 @@ on:
           - eltwise.binary.logical_and.logical_and
           - eltwise.binary.logical_and.logical_and_output
           - eltwise.binary.polyval.polyval
-          - eltwise.binary.remainder.remainder
           - eltwise.binary.squared_difference.squared_difference
           - eltwise.binary.squared_difference_output.squared_difference_output
+          - eltwise.binary.scatter.scatter_forge
+          - eltwise.binary.remainder.remainder
```

Comment:
Where is the associated file to the test?

Reply:
The file is tests/sweep_framework/sweeps/eltwise/binary/remainder/remainder.py
I just grouped the binary remainder tests together for consistency.

Reply:
Got it, missed the fact that it already existed. Thanks